third_party/closure-compiler/externs/web_animations.js

```diff
@@ -27,6 +27,12 @@
 class Animation {
 
   /**
+   * Current time of the animation.
+   * @type {number}
```

Comment:
Yeah. We can't use `time` here, b/c it's also an extern. Even harder, this extern has to go to the Closure's native externs at some point. So, on this level, we can't use `time`. But in our own code, we can.